metric/system/process/process_windows.go

```diff
@@ -201,8 +209,7 @@ func getProcTimes(pid int) (uint64, uint64, uint64, error) {
 
 func procMem(pid int) (uint64, uint64, error) {
 	handle, err := syscall.OpenProcess(
-		windows.PROCESS_QUERY_LIMITED_INFORMATION|
-			windows.PROCESS_VM_READ,
```

Comment:
is `PROCESS_VM_READ` no longer needed anywhere? I don't see it added to a more privileged function call anywhere

Reply:
`PROCESS_VM_READ` is only needed for Windows Server 2003 and Windows XP, we support neither of them.

>A handle to the process. The handle must have the PROCESS_QUERY_INFORMATION or PROCESS_QUERY_LIMITED_INFORMATION access right. For more information, see [Process Security and Access Rights](https://learn.microsoft.com/en-us/windows/desktop/ProcThread/process-security-and-access-rights).
>
>Windows Server 2003 and Windows XP: The handle must have the PROCESS_QUERY_INFORMATION and PROCESS_VM_READ access rights.

https://learn.microsoft.com/en-us/windows/win32/api/psapi/nf-psapi-getprocessmemoryinfo

Reply:
I added a comment explaining it